.github/actions/build-ab/index.js

```diff
@@ -6,43 +6,11 @@ import { S3Client, ListObjectsCommand } from '@aws-sdk/client-s3'
 import { v4 as uuidv4 } from 'uuid'
 import { args } from './args.js'
 import { fetchRetry } from '@newrelic/browser-agent.actions.shared-utils/fetch-retry.js'
+import Handlebars from 'handlebars'
 
-const config = {
-  init: {
-    distributed_tracing: {
-      enabled: true
-    },
-    ajax: {
-      deny_list: [
-        'nr-data.net',
-        'bam.nr-data.net',
-        'staging-bam.nr-data.net',
-        'bam-cell.nr-data.net'
-      ]
-    },
-    session_replay: {
-      enabled: true,
-      sampleRate: 0.5,
-      errorSampleRate: 1
-    }
-  },
+const template = Handlebars.compile(await fs.promises.readFile('./template.js', 'utf-8'))
```

Comment:
If this works, good. I just want to put here that I, personally, would prefer an absolute path to the file being read. It gives me warm and fuzzies that something stupid about the CI/CD environment isn't going to break it.